container_workflow_tool/utility.py

```diff
@@ -51,23 +51,30 @@ def _remove_file(path, logger=None):
         os.remove(path)
 
 
-def _get_packager(config):
+def _get_packager(config) -> str:
     # Try to read which packager to use from the config file, default to rhpkg
     packager = getattr(config, 'packager_util', "fedpkg")
     return packager
 
 
-def _get_hostname_url(config):
+def _get_hostname_url(config) -> str:
     # Try to read which packager to use from the config file
     # default to "https://src.fedoraproject.org"
     git_url = getattr(config, "hostname_url", None)
     return git_url
 
+
 def _split_config_path(config: str) -> (str, str):
-        conf = config.split(':')
-        config_path = conf[0]
-        image_set = conf[1] if len(conf) > 1 else 'current'
-        return (config_path, image_set)
+    conf = config.split(':')
+    if len(conf) > 2:
+        raise RebuilderError(
+            "Config path contains a lot of arguments."
```

Comment:
```suggestion
            "You can only use one image_set with the --config option."
```

Reply:
I would still like to use this suggestion, as this error message is more specific.

Reply:
IT was marked by me as Resolved and I did not get it. Sorry, I will fix it.